From f9664cebaeff842415e292bccdcdb4d47604df05 Mon Sep 17 00:00:00 2001
From: Dante Russo <drusso@codeaurora.org>
Date: Thu, 7 Jul 2016 17:16:55 -0700
Subject: [PATCH 08/11] msm8974: Fix logging level and remove nmea log
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Logging level should be 2 by default to
prevent excessive logging by default.
Nmea log print is removed, as it can be
excessive during gps tracking.

Bugs: 29499503 29272976
Change-Id: Ia60295207b6bbbcb24bbede4f1802c0c7a2f2544
Keep msm8974 in sync with the rest, post removal
Signed-off-by: D. Andrei Măceș <Andrei.Maces@alumni.nd.edu>
---
 msm8974/etc/gps.conf                              | 2 +-
 msm8974/loc_api/libloc_api_50001/loc_eng.cpp      | 1 -
 msm8974/loc_api/libloc_api_50001/loc_eng_nmea.cpp | 1 -
 3 files changed, 1 insertion(+), 3 deletions(-)

diff --git a/msm8974/etc/gps.conf b/msm8974/etc/gps.conf
index 01fee6c..fb8a312 100644
--- a/msm8974/etc/gps.conf
+++ b/msm8974/etc/gps.conf
@@ -21,7 +21,7 @@ NTP_SERVER=time.gpsonextra.net
 # DEBUG LEVELS: 0 - none, 1 - Error, 2 - Warning, 3 - Info
 #               4 - Debug, 5 - Verbose
 # If DEBUG_LEVEL is commented, Android's logging levels will be used
-DEBUG_LEVEL = 3
+DEBUG_LEVEL = 2
 
 # Intermediate position report, 1=enable, 0=disable
 INTERMEDIATE_POS=0
diff --git a/msm8974/loc_api/libloc_api_50001/loc_eng.cpp b/msm8974/loc_api/libloc_api_50001/loc_eng.cpp
index cba5ffa..3c687e8 100644
--- a/msm8974/loc_api/libloc_api_50001/loc_eng.cpp
+++ b/msm8974/loc_api/libloc_api_50001/loc_eng.cpp
@@ -923,7 +923,6 @@ void LocEngReportNmea::proc() const {
     struct timeval tv;
     gettimeofday(&tv, (struct timezone *) NULL);
     int64_t now = tv.tv_sec * 1000LL + tv.tv_usec / 1000;
-    CALLBACK_LOG_CALLFLOW("nmea_cb", %d, mLen);
 
     if (locEng->nmea_cb != NULL)
         locEng->nmea_cb(now, mNmea, mLen);
diff --git a/msm8974/loc_api/libloc_api_50001/loc_eng_nmea.cpp b/msm8974/loc_api/libloc_api_50001/loc_eng_nmea.cpp
index 842eda1..978f7a9 100644
--- a/msm8974/loc_api/libloc_api_50001/loc_eng_nmea.cpp
+++ b/msm8974/loc_api/libloc_api_50001/loc_eng_nmea.cpp
@@ -59,7 +59,6 @@ void loc_eng_nmea_send(char *pNmea, int length, loc_eng_data_s_type *loc_eng_dat
     struct timeval tv;
     gettimeofday(&tv, (struct timezone *) NULL);
     int64_t now = tv.tv_sec * 1000LL + tv.tv_usec / 1000;
-    CALLBACK_LOG_CALLFLOW("nmea_cb", %p, pNmea);
     if (loc_eng_data_p->nmea_cb != NULL)
         loc_eng_data_p->nmea_cb(now, pNmea, length);
     LOC_LOGD("NMEA <%s", pNmea);
-- 
2.17.1

